### ISSUE TITLE
question about two_satge:

### ISSUE THREAD

**Issue by leayz-888** [IDEA-opensource/DAB-DETR#43]
Thank you for your excellent work,
What I want to know is have you used the two-stage strategy when training DAB-Deformable-DETR? For DAB-Deformable-DETR, does it give a performance boost?

**Comment by SlongLiu** [comment]
I believe you will get better results with a two-stage strategy, but we have not this in this paper.

**Comment by leayz-888** [comment]
Thanks for your reply, when I used the two-stage strategy to train my own dataset, I got a much worse result, AP dropped by 4%, the backbone I used was resnext101 (without using pretrained model).
When I use the pre-trained model you provided for fine-tuning, AP drops by 2% compared to Deformable-DETR
May I ask why this happens? Looking forward to your reply.

**Comment by leayz-888** [comment]
> I believe you will get better results with a two-stage strategy, but we did not explore this in this paper.

Thanks for your reply, when I used the two-stage strategy to train my own dataset, I got a much worse result, AP dropped by 4%, the backbone I used was resnext101 (without using pretrained model).
When I use the pre-trained model you provided for fine-tuning, AP drops by 2% compared to Deformable-DETR
May I ask why this happens? Looking forward to your reply.
